cmd/parse_locals.go

```diff
@@ -5,8 +5,8 @@ package cmd
 // parses the `locals` blocks and evaluates their contents.
 
 import (
+	"github.com/gruntwork-io/go-commons/errors"
 	"github.com/gruntwork-io/terragrunt/config"
-	"github.com/gruntwork-io/terragrunt/errors"
```

Comment:
Terragrunt exported that package to the common repository in v0.52.4, I believe.